app/views/components/identity-bar/index.html

```diff
@@ -0,0 +1,73 @@
+{% extends "layouts/base.html" %}
+
+{% block body %}
+
+<h1 class="govuk-heading-xl">
+  <span class="govuk-caption-xl">Components</span> Identity bar
+</h1>
+
+  <h2 class="govuk-heading-m">Standard</h2>
+
+  {{ mojIdentityBar({
+    title: { html: '<b>John Smith</b> and <b>Jane Smith</b> (Financial remedy)' }
+  }) }}
+
+  <br><br>
+
+  <h2 class="govuk-heading-m">With actions</h2>
+
+  {{ mojIdentityBar({
+    title: {
+      html: '<b>John Smith</b> and <b>Jane Smith</b> and <a href="#">4 others</a> (Financial remedy)'
+    },
+    menus: [{
+      items: [{
+        text: 'Do this',
+        href: "#"
+      }, {
+        text: 'Do that',
+        href: "#",
+        classes: 'moj-button--secondary'
```

Comment:
`moj-button--secondary` isn't a _thing_. This should now be the GOV.UK equivalent.